### PR TITLE
More character customization part 1

### PR DESCRIPTION
**Pull request by Asher-42**
## Changelog
added: moth wing (angel)
added: dragon maid tail
added: succubus tail
added: dragon horns long
added: harpy wing arms (big!!)
added: long tail support under modular chomp taur for 64x34
https://i.imgur.com/9P6cCkD.png

https://i.imgur.com/DCH22ML.png

https://i.imgur.com/mGmXjH0.png

made by immovablesaint / magpiemayhem

image

### CODE REVIEW

File: modular_chomp/code/modules/mob/new_player/sprite_accessories_ear.dm [hunk 1]
```diff
@@ -15,4 +15,12 @@
 	extra_overlay = "protogen_horny_fluff"
 	extra_overlay2 = "protogen_horny_horns"
 	do_colouration = 1
-	color_blend_mode = ICON_MULTIPLY
\ No newline at end of file
+	color_blend_mode = ICON_MULTIPLY
+
+/datum/sprite_accessory/ears/Long_dragon_horns_ch
```
Comment (Kashargul) [inline modular_chomp/code/modules/mob/new_player/sprite_accessories_ear.dm:20]: lower case

File: modular_chomp/code/modules/mob/new_player/sprite_accessories_tail.dm [hunk 2]
```diff
@@ -51,3 +51,10 @@
 /datum/sprite_accessory/tail/featheredraptor/twotone
 	name = "feathered Raptor, duel colorable"
 	extra_overlay = "feather_raptor-overlay"
+
+/datum/sprite_accessory/tail/Succubustail_ch
```
Comment (Kashargul) [inline modular_chomp/code/modules/mob/new_player/sprite_accessories_tail.dm:55]: lower case

File: modular_chomp/code/modules/mob/new_player/sprite_accessories_wing.dm [hunk 3]
```diff
@@ -244,3 +249,12 @@
 	do_colouration = 1
 	color_blend_mode = ICON_MULTIPLY
 	wing_offset = 16
+
+/datum/sprite_accessory/wing/Large_Harpy_wings_Ch
```
Comment (Kashargul) [inline modular_chomp/code/modules/mob/new_player/sprite_accessories_wing.dm:253]: lower case

File: modular_chomp/code/modules/mob/new_player/sprite_accessories_taur.dm [hunk 4]
```diff
@@ -334,3 +334,18 @@
...
 	loaf_offset = 5
+
+/datum/sprite_accessory/tail/taur/longtail
+	//hide body parts list() makes it so no limbs are hidden
+	//made its own child so we can reuse this
+	offset_x = -16
+	hide_body_parts = list()
+
+
+/datum/sprite_accessory/tail/taur/longtail/Dragonmaidtail_ch
```
Comment (Kashargul) [inline modular_chomp/code/modules/mob/new_player/sprite_accessories_taur.dm:345]: lower case

**Review by Kashargul — CHANGES_REQUESTED** [review]

File: modular_chomp/code/modules/mob/new_player/sprite_accessories_wing.dm (hunk 3 above)
Comment (Asher-42) [reply modular_chomp/code/modules/mob/new_player/sprite_accessories_wing.dm]: fixed

File: modular_chomp/code/modules/mob/new_player/sprite_accessories_tail.dm (hunk 2 above)
Comment (Asher-42) [reply modular_chomp/code/modules/mob/new_player/sprite_accessories_tail.dm]: fixed

File: modular_chomp/code/modules/mob/new_player/sprite_accessories_ear.dm (hunk 1 above)
Comment (Asher-42) [reply modular_chomp/code/modules/mob/new_player/sprite_accessories_ear.dm]: fixed

**Comment by Asher-42** [comment]
made lower case

**Review by Kashargul — APPROVED** [review]

**Pull request merged**